src/components/BodySection.js

```diff
@@ -611,9 +701,20 @@ export default function BodySection(props) {
           buttonText={genderString}
           optionText={isMale ? femaleString : maleString}
         />
-      </div>
+      </div>  
       <div id="toolTipFront">{drawFront()}</div>
       <div id="toolTipBack">{drawBack()}</div>
+      <svg id="selectionButtonContainer">
+        {createExplainTextNoD3(121, 16, "Get Exercises As", 12, 12, 12, "left")}
+        {createButtonNoD3("unionButton", 12, 20, 30, 30, "DarkSlateGray",
+        selectionButtonClickHandler, null, "default", mouseEnterHandler, mouseLeaveHandler)}
+        {createButtonImage("unionImage", 16, 24,
+        "/icons/union.svg", 22, 1)}
+        {createButtonNoD3("intersectionButton", 12, 55, 30, 30, "white",
+        selectionButtonClickHandler, null, "pointer", mouseEnterHandler, mouseLeaveHandler)}
+        {createButtonImage("intersectionImage", 16, 59,
+        "/icons/intersection.svg", 22, 1)}
+      </svg>

```

Comment:
I think we should start the buttons at the top left with any padding as we do in the circle chart. 
![image](https://user-images.githubusercontent.com/66337956/225863342-92cc0ae3-de50-4782-b103-96791dda9190.png)

Reply:
Yeah the placement of the buttons should be changed. I just wasn't sure if we wanted to put them on the left side or on the right side of the bodies. I think there could be some merit to putting them to the right since it would mean all buttons are closer together (less mouse movement overall). Let me know what you think of this idea

Reply:
Hmm, I personally think that having it on the left side makes the design of each container makes the design consistent. But I'm not opposed to having it on the right as the reasoning you provided makes sense as well. In regards to how it should be aligned, maybe we can add the legend we discussed with the number of muscles the user has selected first and then follow that up with the buttons making the design even more consistent.

Reply:
I think we can stick to having them on the left side of the bodies for now and perhaps re-evaluate later. I'll y-align these buttons with the top-most buttons in the CP chart for now to leave space for a legend to be placed above.